R/make_igv_snapshot.R

```diff
@@ -42,34 +42,34 @@
 #'
 #' @return Path to file (.png).
 #'
-#' @import SRAdb
+#' @import SRAdb GAMBLR.helpers
 #' @export
 #'
 #' @examples
 #' \dontrun{
-#' this_sv = annotated_sv %>% 
+#' this_sv = annotated_sv %>%
 #'  filter(gene=="ETV6")
-#' 
+#'
 #' #you don't need to know the details for the bam file but you can supply it if you want
-#' tumour_bam = get_bams(sample = this_sv$tumour_sample_id)
-#' 
+#' tumour_bam = GAMBLR.results::get_bams(sample = this_sv$tumour_sample_id)
```

Comment:
I don't see this function (yet) in the `GAMBLR.results` package. I am assuming this function is meant to go there. But wanted to flag this in my review.

Reply:
Right, it will need to be added since this is a GSC-tied function. But I will update this example to show how the output of that function looks like and avoid using a non-generally accessible package in example :+1: